analysis_options.yaml

```diff
@@ -5,6 +5,7 @@ analyzer:
 linter:
   rules:
     - avoid_function_literals_in_foreach_calls
+    - avoid_private_typedef_functions
```

Comment:
Ugh...but this lint only applies if there is ONE usage of the typedef. 🤷  sorry

Reply:
ha really? Oh well, it's worth turning on even if its so narrow.